src/server/pfs/cmds/cmds.go

```diff
@@ -1604,13 +1623,15 @@ Objects are a low-level resource and should not be accessed directly by most use
 			}); err != nil {
 				return err
 			}
-			if !errors {
+			if !foundErrors {
 				fmt.Println("No errors found.")
 			}
 			return nil
 		}),
 	}
 	fsck.Flags().BoolVarP(&fix, "fix", "f", false, "Attempt to fix as many issues as possible.")
+	fsck.Flags().BoolVar(&fullZombie, "zombie-all", false, "Check all pipelines for zombie files: files corresponding to old inputs that were not properly deleted")
```

Comment:
Can we have the terminology be consistently `all` or `full`. It seems like `all` would be more consistent with other functionality (i.e. delete pipeline).